src/main/java/net/mcreator/minecraft/MinecraftImageGenerator.java

```diff
@@ -382,6 +382,37 @@ public static BufferedImage generateBrewingPreviewPicture(Workspace workspace, M
 			return icon;
 		}
 
+		/**
+		 * <p>This method generates anvil recipe images.</p>
+		 *
+		 * @param input    Input of the recipe.
+		 * @param addition Addition of the recipe
+		 * @param result   Result of the recipe.
+		 * @return Returns the generated image.
+		 */
+		public static BufferedImage generateAnvilPreviewPicture(Workspace workspace, MItemBlock input,
+				MItemBlock addition, MItemBlock result) {
+			BufferedImage icon = new BufferedImage(28, 28, BufferedImage.TYPE_INT_ARGB);
+			Graphics2D graphics2D = icon.createGraphics();
+			graphics2D.setRenderingHint(RenderingHints.KEY_ANTIALIASING, RenderingHints.VALUE_ANTIALIAS_ON);
+			graphics2D.setColor(new Color(190, 190, 190, 65));
+
+			drawTwoSlotRecipe(graphics2D, workspace, input, addition);
+
+			// TODO: implement hammer drawing (Note from Nerdy: sorry lol)
```

Comment:
Is it something you don't know how to make or did you forget to implement it?

Reply:
I'm leaving this one to Matej as he has done the rest of them

Reply:
@MysteriousWolf, could you take a look? Thanks!